test/terraform_aws_simple_test.go

```diff
@@ -0,0 +1,68 @@
+// =================================================================
+//
+// Work of the U.S. Department of Defense, Defense Digital Service.
+// Released as open source under the MIT License.  See LICENSE file.
+//
+// =================================================================
+
+package test
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/service/lambda"
+	"github.com/stretchr/testify/require"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+)
+
+func TestTerraformSimpleExample(t *testing.T) {
+	t.Parallel()
+
+	region := os.Getenv("AWS_DEFAULT_REGION")
+	require.NotEmpty(t, region, "missing environment variable AWS_DEFAULT_REGION")
+
+	testName := fmt.Sprintf("tt-lf-simple-%s", strings.ToLower(random.UniqueId()))
+
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: "../examples/simple",
+		Vars: map[string]interface{}{
+			"test_name": testName,
+			"tags": map[string]interface{}{
+				"Automation": "Terraform",
+				"Terratest":  "yes",
+				"Test":       "TestTerraformSimpleExample",
+			},
+		},
+		EnvVars: map[string]string{
+			"AWS_DEFAULT_REGION": region,
+		},
+	})
+
+	if os.Getenv("TT_SKIP_DESTROY") != "1" {
+		defer terraform.Destroy(t, terraformOptions)
+	}
+
+	terraform.InitAndApply(t, terraformOptions)
+
+	lambdaFunctionName := terraform.Output(t, terraformOptions, "lambda_function_name")
+	s := session.Must(session.NewSession())
+
+	c := lambda.New(s, aws.NewConfig().WithRegion(region))
+	// https://docs.aws.amazon.com/sdk-for-go/api/service/lambda/#InvokeInput
+	invokeOutput, invokeError := c.Invoke(&lambda.InvokeInput{
+		FunctionName: aws.String(lambdaFunctionName),
+		Payload:      []byte("{}"),
+		// Qualifier:    aws.String("1"),
```

Comment:
Please remove excess/irrelevant comments.

Reply:
@pjdufour-dds done.

Reply:
@chrisgilmerproj @pjdufour-dds I think this PR is ready to be merged?